objectRecognitionSource/src/main/java/edu/ml/tensorflow/Config.java

```diff
@@ -5,6 +5,6 @@
  */
 public interface Config {
     // Params used for image processing
-    public int SIZE = 416; // Edited so that it can be accessed from anywhere
+    public int FRAME_SIZE = 416; // Edited so that it can be accessed from anywhere
```

Comment:
README needs update since we tell folks to change Config.SIZE to match their model